docgen/src/examples/tourism/App.js

```diff
@@ -116,7 +117,7 @@ function HitsMap({hits}) {
     nw: {lat: -85, lng: 180},
     se: {lat: 85, lng: -180},
   });
-  const boundsConfig = fitBounds(boundingPoints, availableSpace);
+  const boundsConfig = hits.length > 0 ? fitBounds(boundingPoints, availableSpace) : {};
```

Comment:
This is due to the <Panel/> feature and the change of behavior when now we are not unmounting widgets anymore. We should warn about the fact that now, people might need to handle empty array of items/hits.